lib/clusteraccess/clusteraccess.go

```diff
@@ -90,6 +93,7 @@ type reconcilerImpl struct {
 	workloadRoleRefs      []commonapi.RoleRef
 	mcpScheme             *runtime.Scheme
 	workloadScheme        *runtime.Scheme
+	SkipWorkloadCLuster   bool
```

Comment:
There is a typo here, but please use the IDE since more lines need to be adjusted.
```suggestion
	SkipWorkloadCluster   bool
```